bandit/blacklists/calls.py

```diff
@@ -343,6 +346,15 @@ def gen_blacklist():
         'Pickle library appears to be in use, possible security issue.'
         ))
 
+    sets.append(utils.build_conf_dict(
```

Comment:
This results in appending a 2nd test name to B301 and we get confusing results where B301 is listed twice.  

The following tests were discovered and loaded:
-----------------------------------------------
	B101	assert_used
	B102	exec_used
	B103	set_bad_file_permissions
	B104	hardcoded_bind_all_interfaces
	B105	hardcoded_password_string
	B106	hardcoded_password_funcarg
	B107	hardcoded_password_default
	B108	hardcoded_tmp_directory
	B110	try_except_pass
	B112	try_except_continue
	B201	flask_debug_true
	B301	dill
	B301	pickle